### DISCUSSION TITLE
Reboot Dinit

### DISCUSSION THREAD

**Discussion by Riolku** [davmac314/dinit#340] [Q&A] [answered]
I see there is `dinitctl shutdown`. Is there an analogous command to reboot the system, short of `kill -INT 1`?

**Comment by mobin-2008** [comment]
Usual Dinit Distribution includes `poweroff`, `reboot`, `halt` and `shutdown` executables for that purpose.

**Reply by Riolku** [reply]
Hmmm then why have the duplication in dinitctl? User daemons?

**Reply by mobin-2008** [reply, marked as answer]
> Hmmm then why have the duplication in dinitctl? User daemons?

Those executables only works when Dinit is PID 1. dinitctl shutdown also works for user daemons.
